package.json

```diff
@@ -54,7 +60,7 @@
   },
   "babel": {
     "presets": [
-      "latest"
+      "env"
```

Comment:
You need to specify here the target node version. Otherwise it'll be the version of node used to do the build (which may not be the target).